freedomofpress/securedrop
# Replace "«" with left chevron PNG icon in "Back to X" navigation links

## Status

Ready for review.

## Description of Changes

Fixes #5596 

Changes proposed in this pull request:

Replace the "«" character in different "Back to X" navigation links with a left chevron PNG icon. The icon is the same as the one already used in the JI (in source breadcrumb) except in the opposite direction.

Alt tag of icons is set to "" so they are ignored by assistive technologies, as per [WCAG recommendations for decorative images](https://www.w3.org/WAI/tutorials/images/decorative/).

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

1. In the Source Interface, navigate to `/why-journalist-key`.
    - [ ] Verify the left chevron icon is present left of the "Back to submission page" link.
1. In the Journalist Interface, access the Admin interface and click on "Add User".
    - [ ] Verify the left chevron icon is present left of the "Back to admin interface" link.
1. From the admin interface, click on the "Edit" button of one of the users.
    - [ ] Verify the left chevron icon is present left of the "Back to admin interface" link.
1. From the admin interface, click on the "Instance Config" button.
    - [ ] Verify the left chevron icon is present left of the "Back to admin interface" link.

Screenshots of expected results are included below for reference.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

## Checklist

### If you made changes to the server application code:


### If you made changes to `securedrop-admin`:


### If you made changes to the system configuration:


### If you made non-trivial code changes:


Choose one of the following:


### If you added or updated a code dependency:

Choose one of the following:
